android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/ReplicatorOfflineTest.java

```diff
@@ -0,0 +1,87 @@
+package com.couchbase.lite;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class ReplicatorOfflineTest extends BaseReplicatorTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testEditReadOnlyConfiguration() throws Exception {
+        Endpoint endpoint = getRemoteTargetEndpoint();
+        ReplicatorConfiguration config = makeConfig(
+                true,
+                false,
+                true,
+                endpoint
+        );
+        config.setContinuous(false);
+        repl = new Replicator(config);
+
+        thrown.expect(IllegalStateException.class);
+        repl.getConfig().setContinuous(true);
+    }
+
+    @Test
+    public void testOfflineStart() throws Exception {
```

Comment:
This test is same as SyncGatewayTest: [testStopReplicatorAfterOffline](https://github.com/couchbase/couchbase-lite-android/blob/34d44d842d378e440519ea73e1d92ee4de63aaa2/android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/ReplicatorWithSyncGatewayTest.java#L68)

Should we copy the same test to `ReplicatorOfflineTest` ?

Reply:
Just leave it at the SyncGatewayTest.

Reply:
I changed my mind. You can copy it here so that we could cover stop()?

Reply:
so we don't want to duplicate same test in two places. even without sync gateway that test can be executed.

Reply:
OK, I think you can move that test here.